testes

```diff
@@ -0,0 +1,70 @@
+#!/bin/fish
```

Comment:
please turn this into a nim (or nims as last resort) file. It's trival to write portable scripts (eg import osproc,os etc)

Reply:
I'm not interested in using NimScript until I can fetch my current working directory (at least).

Reply:
that's why i suggested nim ahead of nimscript; much cleaner than bash/fish etc. 

Plus, I actually want to try your test and would prefer if I dont' have to install fish (plus, windows...). Happy to help if needed.

>  until I can fetch my current working directory (at least).

`echo getCurrentDir()` works in nims; and I've added VM support for `getCurrentDir` + friends in https://github.com/nim-lang/Nim/pull/13813
> We talked about adding getCurrentDir for bump's use-case. I would use this in nimph and testutils and golden and openapi, too.